Adjust Next.js manual setup docs to be in line with wizard (#10156)
getsentry/sentry-docs: docs/platforms/javascript/guides/nextjs/manual-setup.mdx

  // that it will also get attached to your source maps
});
```

```javascript {tabTitle:Edge} {filename:sentry.edge.config.(js|ts)}
import * as Sentry from "@sentry/nextjs";

Sentry.init({
  dsn: "___PUBLIC_DSN___",

  // Set tracesSampleRate to 1.0 to capture 100%
  // of transactions for performance monitoring.
  // We recommend adjusting this value in production
  tracesSampleRate: 1.0,

  // ...

  // Note: if you want to override the automatic release value, do not set a
  // `release` value here - use the environment variable `SENTRY_RELEASE`, so
  // that it will also get attached to your source maps
});
```

We recommend you include your DSN directly in these three files. Alternatively you can pass the DSN via a _public_ environment variable like `NEXT_PUBLIC_SENTRY_DSN`.

While the client initialization code will be injected into your application's client bundle by `withSentryConfig` which we set up earlier,
the configuration for the server and edge runtime needs to be imported from a [Next.js Instrumentation file](https://nextjs.org/docs/app/building-your-application/optimizing/instrumentation).

> You can set this file up by adding a `instrumentation.ts` file to the root directory of your Next.js application (or inside the `src` folder if you are using one) and adding the following content:

Reviewer: ```suggestion
To set up this file, add a `instrumentation.ts` file to the root directory of your Next.js application (or inside the `src` folder if you're using one) and add the following content:
```

> ### Opt Out of Sentry SDK bundling in Client or Server side

Reviewer: ```suggestion
### Opt Out of Sentry SDK bundling in Client- or Server-Side
```

> If you want the `sentry` to be available in your server side & not in client side, you can make your `sentry.client.config.js` empty. This will prevent webpack from pulling in the Sentry related files when generating the browser bundle. The same goes the opposite for opting out of server side bundle by emptying `sentry.server.config.js`.

Reviewer: ```suggestion
If you want the Sentry SDK to be available server-side & not client-side, you can make your `sentry.client.config.js` empty. This will prevent webpack from pulling in the Sentry-related files when generating the browser bundle. Similarly, to opt out of server-side SDK bundling, make your `sentry.server.config.js` empty.
```
